src/Controller/Api/GalleryController.php

```diff
@@ -155,6 +160,8 @@ public function getGalleryAction($id)
      *  }
      * )
      *
+     * @Rest\Get("/galleries/{id}/medias")
```

Comment:
I don't know it the "rest" routing type was creating the same path, but just for the records: https://github.com/sonata-project/SonataMediaBundle/pull/459.

Reply:
I compare this methods with `/api/doc` and set it based on it. I agrre, it should be `media` but change it in minor release is not good idea.